### DISCUSSION TITLE
Capacitor android app build with 'quasar build' not installable

### DISCUSSION THREAD

**Discussion by FragsterAt** [quasarframework/quasar#9516] [CLI - Capacitor / Cordova modes] [answered]
Hello!
I build with apk with ```quasar build -m capacitor -T android```. It create ```app-release-unsigned.apk``` (2388 kb) in ```dist\capacitor...``` folder. Then I copy this APK on real device, try to install and it says: 'program is not installed' (or something like that, i have non-english localization on phone)

Then I run ```quasar build -m capacitor -T android --skip-pkg``` and build apk from android studio, run  it create ```app-debug.apk``` (2863 kb) in ```src-capacitor\android\build...``` folder. When I copy this apk on device - it installs and works.

I want to build apk from one place, what I'm doing wrong?

**Comment by FragsterAt** [comment, marked as answer]
Then I run `quasar build -d -m capacitor -T android`, it creates `app-debug.apk` (3389 kb) in `dist\capacitor...` folder, and it installs and works.

What the difference with this modes? Why 'unsigned' version cant be installed, even if I turn all switches in developer options in phone settings

**Reply by metalsadman** [reply]
@FragsterAt `-d` is short for `--debug` flag, it creates a debug signed apk use for developers, hence why it's installable for testing, while the other creates a `release` version that needs to be signed with a valid key if you want to install it or publish it to the store. https://developer.android.com/studio/publish/app-signing#debug-mode
